packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-textarea/example.html

```diff
@@ -112,16 +112,50 @@ <h2>Examples</h2>
   </fieldset>
 
   <fieldset>
-    <legend>Optional Indicator Mark</legend>
+    <legend>Indicator Mark</legend>
```

Comment:
@katie-bobbe-genesys I don't think there was anything wrong with the component just that the documentation/example was a little confusing.
I have made a change to the `text-area` form field component example and if others agree it is clearer I will duplicate it in the other form field components